VAMobile/documentation/design/Components/Alerts and progress/Snackbar.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Snackbar
+---
+
+Snackbars provide feedback regarding API interactions at the bottom of the screen.
+
+## Examples
+
+### Default
+**Open in**: [Storybook](https://department-of-veterans-affairs.github.io/va-mobile-library/?path=/docs/snackbar--default)  |   [Figma](https://www.figma.com/design/Zzt8z60hCtdEzXx2GFWghH/%F0%9F%93%90-Component-Library---Design-System---VA-Mobile?node-id=263-702&t=1BgtKDvOeoxfzmzR-4)
+<iframe width="620" height="100" alt="Image of component in Storybook" src="https://department-of-veterans-affairs.github.io/va-mobile-library/?path=/story/snackbar--default&full=1&shortcuts=false&singleStory=true" allowfullscreen></iframe>
+
+### Variations
+**Open in**: [Storybook](https://department-of-veterans-affairs.github.io/va-mobile-library/?path=/docs/snackbar--with-action)  |   [Figma](https://www.figma.com/design/Zzt8z60hCtdEzXx2GFWghH/%F0%9F%93%90-Component-Library---Design-System---VA-Mobile?node-id=263-702&t=1BgtKDvOeoxfzmzR-4)
+<iframe width="620" height="100" alt="Image of component in Storybook" src="https://department-of-veterans-affairs.github.io/va-mobile-library/?path=/story/snackbar--with-action&full=1&shortcuts=false&singleStory=true" allowfullscreen></iframe>
+
+## Usage
+
+### When to use Snackbar
+* **If a user action that triggers an API call is successful or results in an error.** The snackbar may allow users to take an action on the feedback such as trying again or undoing the action. 
+
+### How this component works
+- The **icon** will be relevant to the state of the snackbar (success or error).
+- The **text content** is a short description of the success or error action. It is used to inform the status of an action.
+- The **left action button** is optional and should only be used when there is an additional action for the user to take on the snackbar. The left action button can be an "Undo" or "Try again" button. 
+     - The undo button allows a user to reverse a successful action. When the action is successfully reversed, another snackbar will open informing the successful undo. The new snackbar will only have the dismiss button.
+     - The try again button allows a user to initiate a failed action. This will prompt the system to redo the action and either produce a successful or unsuccessful snackbar.
+- The **right action button** is a dismiss button that allows a user to manually close the snackbar.
+
+### Behavior
+- The current iteration of the snackbar will only be dismissible by the user selecting the “Dismiss” button, another snackbar opening, or the user navigating to a different screen. Additional iterations may incorporate a user-defined timebox on the snackbar.
```

Comment:
Not sure if it falls here or below in the accessibility, but we do auto-dismiss non-actionable (no Undo/Try again) Snackbars after 5 seconds if the screen reader is on per discussion with Brea